feconf.py

```diff
@@ -566,6 +566,7 @@ def get_empty_ratings():
 SIGNUP_DATA_URL = '/signuphandler/data'
 SIGNUP_URL = '/signup'
 SPLASH_URL = '/splash'
+STORE_TRAINED_CLASSIFIER_MODEL_HANDLER = '/ml/trainedclassifierhandler'
```

Comment:
Can we call it just TRAINED_CLASSIFIER_MODEL_HANDLER (similar to the name of the controller)?